include/gmsh.h

```diff
@@ -5,12 +5,17 @@
 #include <fstream>
 #include <iostream>
 #include <limits>
+#include <map>
 #include <memory>
 #include <vector>
-
 #include "mesh.h"
 #include "point.h"
 
+//libraries
+#include <Eigen/Dense>
+using Eigen::MatrixXd;
```

Comment:
Avoid shortening of namespace, use `Eigen::MatrixXd` everywhere to avoid namespace clash.